clang/test/SemaCXX/cxx2c-binding-pack.cpp

```diff
@@ -124,6 +125,14 @@ void lambda_capture() {
   [&x...] { (void)sum(x...); }();
 }
 
+struct S2 {
+    int a, b, c;
+};
+
+auto X = [] <typename = void> () {
```

Comment:
I feel like there are a bunch of varieties on this case we can test as well: https://godbolt.org/z/r9zxxfoW5

Reply:
Oh, sorry, I added your test with `clsss` not realizing you were talking about modifying an existing test. :sweat_smile: